.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+  generate-winmd:
+    name: Generate winmd
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Clean
+        run: Remove-Item -Recurse -Force .windows/winmd/
+      - uses: ilammy/msvc-dev-cmd@v1 # Needed to find CL.EXE
```

Comment:
The third-party dep. is costly, won't work for Microsoft repos, etc. Perhaps replace it with https://github.com/microsoft/windows-rs/blob/master/.github/workflows/test.yml#L63-L74 instead. Those paths are guaranteed.

Reply:
Thanks! That's the kind of info I was looking for :)

Reply:
https://github.com/MarijnS95/washington-rs/actions/runs/3600429985/jobs/6065073771 looks like I need to lock in to `windows-2022` and update this to the latest versions: https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md#installed-windows-sdks

Reply:
Apologies for all the repeated pushes, looks like it's still "early" :sleeping:

Reply:
https://github.com/MarijnS95/washington-rs/actions/runs/3600562233/jobs/6065337365 and success, thanks!